client/src/components/layouts/DashboardSidebarData.js

```diff
@@ -26,7 +26,7 @@ const DashboardSidebarData = ({ setLogoutAlert }) => {
       link: '/dashboard',
     },
     {
-      title: 'Our Homepage',
+      title: 'Homepage',
```

Comment:
The reason I use `Our Homepage` is to distinguish between `Researchify homepage` and `User's homepage`, so that user won't click on it to navigate to `Researchify homepage`. Either is fine for me though, what do you guys think?

Reply:
Also prefer `Our Homepage` until we figure out a better naming

Reply:
yeah that makes sense